java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -1933,15 +1933,6 @@ public void testSetInitialCapacity() {
       vector.allocateNew();
       assertEquals(4096, vector.getValueCapacity());
       assertEquals(64, vector.getDataBuffer().capacity());
-
-      boolean error = false;
-      try {
-        vector.setInitialCapacity(5, 0.1);
```

Comment:
same here

Reply:
We take a max and set it to 1 if needed. Exception handling is not needed.

Reply:
added back the test with assertion